gluefactory/datasets/rdnim.py

```diff
@@ -77,7 +78,20 @@ def _init(self, conf):
 
     def _read_image(self, idx: int, type: str) -> dict:
         img = load_image(self._files[idx][type], self.conf.grayscale)
-        return self.preprocessor(img)
+
+        # get mask for black pixels in the image to identify synthetic borders
+        np_img = read_image(self._files[idx][type], self.conf.grayscale)
+        mask = ~np.all(np_img == 0, axis=2)
```

Comment:
Can we rename the mask variable? It's a bit unexpressive

Reply:
Done, changed to `padding_mask`